### ISSUE TITLE
DL3066 false positive

### ISSUE THREAD

**Issue by firefart** [hadolint/hadolint#1234]
I currently have this in my dockerfile

```docker
ARG UID=1000
ARG GID=1000
....
USER ${UID}:${GID}
```

And the latest version of hadolint prints a DL3066 on the `USER` line although the 2 parameters are numeric.

**Comment by guest20** [comment]
#1222

**Comment by firefart** [comment]
Sorry, only had a look at the open issues
